static/css/ubuntu.css

```diff
@@ -1,27 +1,28 @@
 @font-face {
     font-family: 'Ubuntu';
-    src:url('../fonts/Ubuntu-Regular.woff2') format('woff2');
+    src: url('https://fonts.gstatic.com/s/ubuntu/v20/4iCv6KVjbNBYlgoCxCvjsGyNPYZvgw.woff2')
+        format('woff2');
     font-display: swap;
     font-weight: 100 400;
 }
 
 @font-face {
     font-family: 'Ubuntu';
-    src:url('../fonts/Ubuntu-Bold.woff2') format('woff2');
+    src: url('../fonts/Ubuntu-Bold.woff2') format('woff2');
     font-display: swap;
     font-weight: 800;
 }
 
 @font-face {
     font-family: 'Ubuntu';
-    src:url('../fonts/Ubuntu-Light.woff2') format('woff2');
+    src: url('../fonts/Ubuntu-Light.woff2') format('woff2');
     font-display: swap;
     font-weight: 200;
 }
 
 @font-face {
     font-family: 'Ubuntu';
-    src:url('../fonts/Ubuntu-Medium.woff2') format('woff2');
+    src: url('../fonts/Ubuntu-Medium.woff2') format('woff2');
     font-display: swap;
     font-weight: 500;
 }
```

Comment:
Why do we have some font-faces not from google?

Reply:
I think you are viewing the outdated file.Can you check the new commit for the file

Reply:
Perfect, thanks! Just walked through commits )